### PR TITLE
Expand 'sustained disruption of discussion' text to explicitly call out concern trolling, sealioning, gish galloping, and argumentum ad nauseam.

### LINKED ISSUE #150: Should there be explicit language in the CEPC Regarding "Sea Lioning"?
W3C process only functions smoothly when people engage in discussions in good faith. In some cases, people engaging in W3C discussions use the w3c process in order to stall discussion, sow confusion, or sabotage some existing work. Should we have some explicit language in the CEPC to help members of our community handle these types of situations?

### PR DESCRIPTION
**Pull request by hober**
An attempt at addressing #150.

***
<a href="https://pr-preview.s3.amazonaws.com/hober/PWETF/pull/171.html" title="Last updated on Jun 2, 2021, 4:29 PM UTC (89e26bb)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/PWETF/171/8ce145e...hober:89e26bb.html" title="Last updated on Jun 2, 2021, 4:29 PM UTC (89e26bb)">Diff</a>

### CODE REVIEW

**Review by tantek — APPROVED** [review]

**Review by wareid — APPROVED** [review]

**Comment by jwrosewell** [comment]
This modification is an attempt to stifle debate and increase the imbalance of power between group chairs and participants. In particular it discriminates against new members who can not understand from the documents how a group reached a position. It discourages diversity by promoting group think. If a group is insufficiently confident in its position to engage in debate supported by evidence, then its position is likely in need of modification rather than the PWETF.

As an example. Issue “[Supply chains can be trusted - expand document to consider this possibility](https://github.com/w3ctag/security-questionnaire/issues/83)” in relation to the Security and Privacy Questionnaire is likely to be considered by the proposer and supporters of this change to be illustrative of some of the issues they perceive need addressing. TAG have done nothing to justify the position of that document, and then banned me without justification as I was progressing a pull request to that document. I believe this pull request is an attempt to retrospectively provide the justification they seek.

In relation to the text of this pull request, I have the following initial questions and concerns.

1.	How does the “bad faith” label get assigned? Is it just the assumption of a motive that the chair considers to be in some way incompatible with their definition of “good”? I don’t have to agree with someone to accept that from their position they are debating in good faith. 
2.	How do we prevent the PWETF from being used to intimidate new members who lack the history of a document and can’t find the evidence to support the position of the group but need such information to engage in the debate or understand the proposal? Without such information a new participant can not propose changes to a document from a position of understanding.
3.	How is motive assigned to a question? If someone can not understand the position of the group, then they will be discouraged from asking to find the information they need. They will retract and become disenfranchised. 
4.	If an argument is weak then the group should be easily able to expose it as such quickly. What it the group’s position is exposed as weak but the chairs and long-standing group participants are committed to maintaining it?

If the issue is use of people’s time and a concern that new participants questions may take time from the group or chairs to resolve, then there are other ways of handling this. These include.

1.	Ensuring all documents have supporting evidence and a background section that summarises the history of the document and its position. Many W3C documents do not contain such information and operate from the position of “everyone knows”.
2.	Where there are contentious issues that lack consensus, then drop them. As a consensus-based organisation without consensus we can’t move forward. As W3C participation increases and more diverse views are advanced then this many be hard for long time W3Cs to accept, but it is the reality of the W3C’s model. More time will be needed for debate.

I would much prefer TAG members focused their talents on providing the evidence needed for new participants to understand their position than avoid debate by making discriminatory proposals such as this one.

I would prefer W3C focused on changes to increase participation and break out of the current group think bubble. The web is for everyone.

File: index.html [hunk 1]
```diff
@@ -262,7 +262,30 @@ <h3 id="unacceptablebehavior">
             <a>Harassing</a> photography or recording, including logging online
             activity for <a>harassment</a> purposes.
           </li>
-          <li>Sustained disruption of discussion.
+          <li>Sustained disruption of discussion. This may include, but
+          is not lmited to, various common methods of engaging in bad
```
Comment (TzviyaSiegman) [inline index.html:266]: s/lmited/limited

Comment (TzviyaSiegman) [reply index.html:266]: Thank you so much @hober. This is extremely helpful!

**Review by torgo — APPROVED** [review]
Fully support these additions.

**Comment by TzviyaSiegman** [comment]
@jwrosewell thanks for your questions. As with all matters in a code of conduct, we have to find the right balance between trusting those who implement the code to interpret it and over-defining it so that it is too constrained to be meaningful. When identifying unacceptable behaviors among humans, it is often situational. That is why terms like "bad faith" are used. We obviously encourage new participants to ask questions and get up to speed. 
The goal of this addition is to discourage people who intentionally slow down progress by doing things like asking unnecessary questions, which takes time away from building implementations or writing spec language. We are open to revising the language if you have suggestions.

**Review by AdaRoseCannon — APPROVED** [review]
This makes sense to me, I do not share the concerns about it being used to prevent new people from becoming engaged since the PWETF is applied by people who can make judgement calls about when a participant is being genuinely inquisitive or disruptive. 

Especially since common forms of disruption happen under the guise of 'just asking questions' or 'just having concerns' where both parties know it is trolling but the person being disruptive can pretend to be sincere to a casual onlooker.

**Comment by jwrosewell** [comment]
@TzviyaSiegman I will propose modifications next week as you suggest. In the meantime I ask that you do not merge this pull request until I have been given the opportunity to do so.

File: index.html (hunk 1 above)
Comment (hober) [reply index.html]: Fixed in 5ce7e84.

**Review by tantek — APPROVED** [review]

**Review by TzviyaSiegman — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -262,7 +262,30 @@ <h3 id="unacceptablebehavior">
...
+              to provoke a critical response in order to appear a victim
+              (<dfn>sealioning</dfn>),
+              </li>
+              <!-- https://rationalwiki.org/wiki/Gish_Gallop -->
+              <li>overwhelming a debate with many weak arguments in an
+              attempt to cause others to waste time refuting them
+              (<dfn>Gish Galloping</dfn>),
+              </li>
+              <!-- https://rationalwiki.org/wiki/Argumentum_ad_nauseam -->
+              <li>or repeatedly making claims already shown to be false
```
Comment (martinthomson) [inline index.html:285]:
```suggestion
              (<dfn>Gish Galloping</dfn>), or
              </li>
              <!-- https://rationalwiki.org/wiki/Argumentum_ad_nauseam -->
              <li>repeatedly making claims already shown to be false
```

**Review by martinthomson — APPROVED** [review]
LGTM

**Review by tantek — APPROVED** [review]

**Comment by jwrosewell** [comment]
@hober I have created a PR which reduces the proposed addition to the following.

- Wasting others time.
 - Repeating statements which have been shown to be false.
 - Re-raising questions that have already been answered.

This has the following advantages.

1. Focus on the outcome which is the wasting of others time.
2. Removed the references to "concern trolling", "sealioning”" "gish galloping", and "ad nauseam" as these are all terms that are not common in plain English and require the reader to research them to understand the document requiring unnecessary time on the part of the reader. They risk also being defined by reference which I would prefer to avoid. 
3. The terms will be harder to understand for those where English is not their first language. 
4. I have tested both the original text and the proposed replacement at [https://app.readable.com/text/?demo](https://app.readable.com/text/?demo). The original text is ranked as D and the replacement A.
5. A weak argument is easy to dismiss otherwise it would not be weak and therefore is covered by "shown to be false".
6. The "basic concepts" statement is concerning as it would discourage participation. The charter for a group should explain the expected level of understanding of the participants and provide guidance on any prior reading or history to help them. If the participant has not read this material then the chair can bring this to their attention.
7. Motive is hard to determine and therefore statements like "disingenuously", "attempt" or "bad faith" need to be avoided. Raising the same question can easily be identified based on a simple analysis of the question. Clarifying questions should be welcomed.

**Review by rhiaro — APPROVED** [review]
